monix-reactive/shared/src/test/scala/monix/reactive/internal/builders/CharsReaderObservableSuite.scala

```diff
@@ -56,6 +58,20 @@ object CharsReaderObservableSuite extends SimpleTestSuite {
     assert(s.state.tasks.isEmpty, "should be left with no pending tasks")
   }
 
+  test("fromCharsReaderUnsafe should throw if the chunkSize is zero or negative") {
```

Comment:
I created two similar (kind-of) tests, one for Unsafe and one for safe. This is because calling `fromCharsReaderUnsafe` throws the error directly, while using `fromCharsReader` will signal via `onError`. If you think the test is redundant, please advise which one to keep.